static/style.css

```
@@ -100,3 +113,36 @@ img[alt="XMR Logo"] {
 @media (min-width: 100em) {
 	#artlist { column-count: 3 ;}
 }
+
+
+input#search {
+  all: unset;
+  background: #222;
+  color: #fff;
+  padding: 0.7rem 1rem;
+  border-radius: 5px;
+  width: 100%;
+}
+
+.search {
+  width: 400px;
+  max-width: 85vw;
+  position: relative;
+  margin: 0.5rem auto 1.2rem;
+  display: flex;
+}
+
+button.clear-search {
+  all: unset;
+  position: absolute;
+  right: 4px;
+  top: 5px;
+  height: 30px;
+  width: 30px;
+  color: #888;
+  cursor: pointer;
+  transition: color 180ms ease-in-out;
+}
+button.clear-search:hover {
+  color: #eee;
+}
```

```suggestion
}

.matched-recipe {
  font-size: 1.5em;
}
```

Add CSS for the matched items. Can be whatever.